<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border-width: 1px;
	border-style: solid;
	margin-top:10px
}

QGroupBox::title {
	font: 14px &quot;Microsoft YaHei&quot;;
	color: #000000;
	subcontrol-origin: margin;
	left:20px;
	padding: 0px 5px 0px 5px;
}</string>
    </property>
    <property name="title">
     <string>文法分析</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="grammarInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border-width: 1px;
	border-style: solid;
	margin-top:10px
}

QGroupBox::title {
	font: 14px &quot;Microsoft YaHei&quot;;
	color: #000000;
	subcontrol-origin: margin;
	left:20px;
	padding: 0px 5px 0px 5px;
}</string>
    </property>
    <property name="title">
     <string>句子分析</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="sentenceInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>841</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border-width: 1px;
	border-style: solid;
	margin-top:10px
}

QGroupBox::title {
	font: 14px &quot;Microsoft YaHei&quot;;
	color: #000000;
	subcontrol-origin: margin;
	left:20px;
	padding: 0px 5px 0px 5px;
}</string>
    </property>
    <property name="title">
     <string>文法分析结果</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>按钮</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="anaBtn">
          <property name="text">
           <string>分析文法</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="firstBtn">
          <property name="text">
           <string>展示First/Follow集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dfaBtn">
          <property name="text">
           <string>展示DFA图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="judgeBtn">
          <property name="text">
           <string>判断文法</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tableBtn">
          <property name="text">
           <string>展示SLR(1)分析表</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="outputTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>841</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border-width: 1px;
	border-style: solid;
	margin-top:10px
}

QGroupBox::title {
	font: 14px &quot;Microsoft YaHei&quot;;
	color: #000000;
	subcontrol-origin: margin;
	left:20px;
	padding: 0px 5px 0px 5px;
}</string>
    </property>
    <property name="title">
     <string>分析栈</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="sentenceBtn">
       <property name="text">
        <string>分析句子</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="stackTable"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
